vector/src/slice.rs

```diff
@@ -114,6 +114,20 @@ where
         self.start.checked_add(idx).and_then(|i| self.vec.get(i))
     }
 
+    /// Gets an element at a given index, panicking if `idx` is out-of-bounds.
```

Comment:
The documentation is out of sync with the actual method (get vs set)

```suggestion
    /// Sets an element at a given index, panicking if `idx` is out-of-bounds.
```